src/pages/reviewed-preprints/[msid]/[...tab].page.tsx

```diff
@@ -27,24 +28,38 @@ const getPublishedDate = (events: TimelineEvent[]): string | undefined => {
   return undefined;
 };
 
-export const Page = (props: PageProps): JSX.Element => (
-  <>
-  <Head>
-    <title>{contentToText(props.metaData.title)}</title>
-    <meta name="citation_title" content={contentToText(props.metaData.title)}/>
-    <meta name="citation_publisher" content="eLife Sciences Publications Limited"/>
-    <meta name="citation_doi" content={props.metaData.doi}/>
-    <meta name="citation_publication_date" content={getPublishedDate(props.status.timeline)}/>
-    <meta name="citation_pdf_url" content={props.metaData.pdfUrl}/>
-    <meta name="citation_fulltext_html_url" content={`https://elifesciences.org/reviewed-preprints/${props.metaData.msid}`}/>
-    <meta name="citation_language" content="text/html"/>
-    { props.metaData.authors.map((author, index) => <meta key={index} name="citation_author" content={`${author.givenNames?.join(' ')} ${author.familyNames?.join(' ')}`}/>)}
-  </Head>
-  <ArticlePage metaData={props.metaData} msidWithVersion={props.msidWithVersion} status={props.status} activeTab="fulltext">
-    <ArticleFullTextTab content={props.content} metaData={props.metaData} peerReview={props.peerReview}></ArticleFullTextTab>
-  </ArticlePage>
-  </>
-);
+export const Page = (props: PageProps): JSX.Element => {
+  const tabs: { [key: string]: () => JSX.Element } = {
```

Comment:
I'm not sure if/how Next would export this into client side JavaScript, maybe this is worth lifting out of the function component? (just a couple of lines above)

Reply:
This was just to make the typescript compiler happy - delivered to the client it's removed during the compile step (as all types are). I'm happy to move it, but I'm not sure it's adding much as a standalone type.